dkms.in

```diff
@@ -864,6 +872,9 @@ prepare_kernel()
 
 prepare_signing()
 {
+   # Lazy source in signing related configuration
+   read_framework_conf $dkms_framework_signing_variables
```

Comment:
Nit: quotes around the argument - here and below

Reply:
`dkms_framework_signing_variables` is a string containing multiple variable names. `dkms_framework_signing_variables="sign_file mok_signing_key mok_certificate"` 
And `read_framework_conf` accepts multiple variable names as arguments. So there should be no quotation marks here.

Reply:
Right we're using `"$@"` in the function - I've misread that as `$1`